src/Workers/Indexers/Outputs.ts

```diff
@@ -24,6 +25,7 @@ export const outputIndexer: IndexHandler = {
           txid: vout.txid,
           n: vout.n,
         },
+        scriptPubKey: vout.scriptPubKey,
```

Comment:
This was used in Runes.GetSpendablesIndex method, since we have the fixed Address.GetSpendables and Address.GetUnspents with runes support this Runes method is not really needed anymore.